### PR TITLE
Add json api to homepage

### PR DESCRIPTION
**Pull request by davidthomas22**
### Context

> Does this issue have a Trello card?

https://trello.com/c/OQudbIvA

### Intent

> What changes are introduced by this PR that correspond to the above card?

Enable JSON API for homepage

### Checklist

- [ ] This PR contains **only** changes related to the above card
- [ ] Tests have been added/updated to cover the change
- [ ] Documentation has been updated where appropriate
- [ ] Tested in Development

### CODE REVIEW

File: server/__tests__/app.spec.js [hunk 1]
```diff
@@ -101,32 +94,21 @@ describe('App', () => {
...
-        );
-        expect(response.text).not.toContain(
-          error.stack,
-          'it should not show the stack',
+          'Could not determine establishment',
         );
       });
   });
 
   it('shows the stack trace on error pages', async () => {
```
Comment (jonhindle) [inline server/__tests__/app.spec.js:104]: The test description doesn't really reflect the test.

File: server/__tests__/app.spec.js [hunk 2]
```diff
@@ -83,16 +83,9 @@ describe('App', () => {
 
   it('hides the stack trace on error pages', async () => {
```
Comment (jonhindle) [inline server/__tests__/app.spec.js:84]: The test description doesn't really reflect the test anymore.

File: server/repositories/cmsQueries/__tests__/HomepageQuery.spec.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        '/jsonapi/prison/berwyn/node/featured_articles?include=field_moj_featured_tile_large.field_moj_thumbnail_image%2Cfield_moj_featured_tile_large.field_image%2Cfield_moj_featured_tile_small.field_moj_thumbnail_image%2Cfield_moj_featured_tile_small.field_image&page%5Blimit%5D=1&fields%5Bnode--featured_articles%5D=title%2Cdrupal_internal__nid%2Cfield_moj_featured_tile_large%2Cfield_moj_featured_tile_small&fields%5Bnode--page%5D=drupal_internal__nid%2Cfield_moj_thumbnail_image%2Cfield_image%2Ctitle%2Cfield_moj_description%2Cfield_moj_series&fields%5Bnode--moj_video_item%5D=drupal_internal__nid%2Cfield_moj_thumbnail_image%2Cfield_image%2Ctitle%2Cfield_moj_description%2Cfield_moj_series&fields%5Bnode--moj_radio_item%5D=drupal_internal__nid%2Cfield_moj_thumbnail_image%2Cfield_image%2Ctitle%2Cfield_moj_description%2Cfield_moj_series&fields%5Bnode--moj_pdf_item%5D=drupal_internal__nid%2Cfield_moj_thumbnail_image%2Cfield_image%2Ctitle%2Cfield_moj_description%2Cfield_moj_series&fields%5Bnode--landing_page%5D=drupal_internal__nid%2Cfield_moj_thumbnail_image%2Cfield_image%2Ctitle%2Cfield_moj_description%2Cfield_moj_series&fields%5Bfile--file%5D=drupal_internal__fid%2Cid%2Curi',
```
Comment (jonhindle) [inline server/repositories/cmsQueries/__tests__/HomepageQuery.spec.js:8]: This feels a bit of a copout test? Although equally testing `addFields` calls feels a bit like testing the code not the functionality? Meh, I guess it shows query hasn't changed 🤷‍♀️

Comment (andrewrlee) [reply server/repositories/cmsQueries/__tests__/HomepageQuery.spec.js:8]: as discussed!

File: server/services/__tests__/cmsService.spec.js [hunk 4]
```diff
@@ -0,0 +1,145 @@
+const {
+  HomepageQuery,
+} = require('../../repositories/cmsQueries/homepageQuery');
+const { TopicsQuery } = require('../../repositories/cmsQueries/topicsQuery');
+const { CmsService } = require('../cms');
+
+describe('Topics Service', () => {
```
Comment (andrewrlee) [inline server/services/__tests__/cmsService.spec.js:7]: Cms Service

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**